napari/components/_tests/test_viewer_model.py

```diff
@@ -307,6 +307,8 @@ def test_swappable_dims():
 
     labels_data = np.random.randint(20, size=(7, 12, 10, 15))
     labels_name = viewer.add_labels(labels_data).name
+    for i in range(labels_data.ndim):
+        viewer.dims.set_point(i, 0)
```

Comment:
I'm not sure if I should fix this test in this way, or change somehow reset function?